### PR TITLE
Cleanup of the enterprise catalog views

### PR DESCRIPTION
**Pull request by alex-sheehan-edx**
## Description

Breaking down the catalog api views into their own files for clarity.

## Post-review

Squash commits into discrete sets of changes

### CODE REVIEW

**Review by kiram15 — APPROVED** [review]
Wow this looks so much better than a 717 line file .... woof

File: enterprise_catalog/apps/api/v1/views/enterprise_cusomter.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import logging
```
Comment (binodpant) [inline enterprise_catalog/apps/api/v1/views/enterprise_cusomter.py]: typo in filename 
`enterprise_catalog/apps/api/v1/views/enterprise_cusomter.py`

**Review by binodpant — APPROVED** [review]
as long as the tests all pass and there is no actual url change this looks fine to me,

I commented on one typo you want to fix

please ensure the routes are all functional. I hope we have sufficient test cases to cover all the routes

File: enterprise_catalog/apps/api/v1/views/enterprise_cusomter.py (hunk 1 above)
Comment (alex-sheehan-edx) [reply enterprise_catalog/apps/api/v1/views/enterprise_cusomter.py]: yikes!

**Pull request merged**